OracleTuxedo/ART_SALT_TMA/dockerfiles/README.md

```diff
@@ -0,0 +1,21 @@
+# Introduction
+This folder contains the information and examples of how to use [Tuxedo](http://oracle.com/tuxedo) with [Docker](https://www.docker.com/).
+
+## Contents
+It is based on the WebLogic Server dockerization (is that even a word?) done by Bruno Borges.
+
+## To use
+1. Into an empty directory:
+  1. Download the Tuxedo TMA 12.2.2 Linux 64 bit installer from OTN
+  2. Download TuxedoARTTMA.zip from this github directory
+  3. Download the latest Tuxedo rolling patch from My Oracle Support, not less than RP003
```

Comment:
You cannot require MOS access for publicly accessible Dockerfiles. If a user without Oracle support is unable to downloading this rolling patch, we cannot accept this PR.